evaluation.py

```diff
@@ -659,6 +659,103 @@ def run_ancestor_properties(args):
     plt.clf()
 
 
+def imputation_accuracy_worker(args):
+    simulation_args, missing_proportion = args
+    ts = msprime.simulate(**simulation_args)
+    np.random.seed(simulation_args["random_seed"])
+    G = ts.genotype_matrix()
+    missing = np.random.rand(ts.num_sites, ts.num_samples) < missing_proportion
+    G[missing] = tskit.MISSING_DATA
+    with tsinfer.SampleData(ts.sequence_length) as sample_data:
+        for var in ts.variants():
+            sample_data.add_site(
+                var.site.position, alleles=var.alleles, genotypes=G[var.site.id]
+            )
+
+    # FIXME need to turn off path compression when running this with n=1000
```

Comment:
This is weird? Is this because of the recent changes that convert time to freq rather than to counts? I think the PC_ANCESTOR_INCREMENT value is probably too large now, which might be at the root of this problem (see also https://github.com/tskit-dev/tsinfer/issues/210#issuecomment-553875914

Reply:
Yes, that's the issue all right. I'll follow up on it.